runtime/dart_init.cc

```diff
@@ -243,6 +244,7 @@ Dart_Isolate IsolateCreateCallback(const char* script_uri,
   // Are we running from a Dart source file?
```

Comment:
The service isolate is still being started from source, the old way? It has not been migrated to using the kernel platform file?